static/js/tools/map/d3_map.tsx

```diff
@@ -176,23 +176,23 @@ export function D3Map(props: D3MapProps): JSX.Element {
       zoomDcid,
       zoomParams
     );
-    if (display.value.showMapPoints && props.mapPointValues) {
+    if (display.value.showMapPoints) {
       let mapPointSvTitle = "";
       if (statVar.value.mapPointSv !== statVar.value.dcid) {
         mapPointSvTitle =
           statVar.value.mapPointSv in statVar.value.info
             ? statVar.value.info[statVar.value.mapPointSv].title
             : "";
       }
-      const filteredMapPoints = props.mapPoints.filter((point) => {
-        return point.placeDcid in props.mapPointValues;
-      });
```

Comment:
When mapPointValues exist (i.e., mapsv) should we still filter?

Reply:
sure, updated!

Reply:
chatted with Bo about this & we think for consistency, it'll make more sense to show all the points & just show "data unavailable" for points missing data. Also when mapPointValues exist, there aren't going to be a lot of points without data, so it doesn't look too different from before.

Reply:
ok, sg